### PR TITLE
Use oc-resource instead of plain text

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7205

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-pdf-viewer/src/App.vue [hunk 1]
```diff
@@ -57,6 +55,8 @@ export default {
         this.loading = true
         const response = await this.getFileContents(fileContext.path, { responseType: 'blob' })
         this.blobUrl = URL.createObjectURL(response.body)
+        const fileInfo = await this.getFileInfo(this.currentFileContext.path, {})
```
Comment (kulmann) [inline packages/web-app-pdf-viewer/src/App.vue:58]:
```suggestion
        const fileInfo = await this.getFileInfo(fileContext.path, {})
```
fileContext is already in the params of `loadPdf`

**Review by kulmann — APPROVED** [review]
so nice 😍

**Comment by kulmann** [comment]
Additional enhancement for https://github.com/owncloud/web/issues/7198

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/26598/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/26598/tracing/file-alice-2022-7-1-01-17-42.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/26598/tracing/file-brian-2022-7-1-01-17-51.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/26598/tracing/public-link-alice-2022-7-1-01-15-45.zip 


</p></details>

**Pull request merged**